src/lib/syncable_ledger/test_mask.ml

```diff
@@ -0,0 +1,108 @@
+open Core
+
+(* Input for testing the interaction between merkle *)
+
+module Make (Input : sig
+  val depth : int
+
+  val mask_layers : int
+end) =
+struct
+  open Merkle_ledger_tests.Test_stubs
+
+  module Hash = struct
```

Comment:
Except for `to_hash`, this module looks to be the same as the `Hash` in the test stubs, and `to_hash` doesn't appear to be used. Could you just use the `Hash` in the test stubs?